locales/templates/app.pot

```diff
@@ -218,6 +245,7 @@ msgctxt "menu_action_rename"
 msgid "Rename"
 msgstr ""
 
+#. Removed to align with dialogue_collecitioninfo_title_info
```

Comment:
@flodolo Sorry for the misunderstanding. The comment was for engineer. @benjamin-cheng Since we don't need this string, I was suggesting removing from the code.

Reply:
@ChenMorpheus it's not referenced in the code anymore, so feel free to remove the string from file.